python/mlx/nn/losses.py

```diff
@@ -1,12 +1,14 @@
 # Copyright © 2023 Apple Inc.
 
 import math
+from typing import Literal, TypeAlias
 
 import mlx.core as mx
-from mlx.nn.layers.base import Module
 
+Reduction: TypeAlias = Literal["none", "mean", "sum"]
```

Comment:
This is really nice! And sadly it's 3.11 and up. So do you mind removing it since we currently support 3.8 and up?

Reply:
I removed the `TypeAlias` and tested it in a Python 3.8 environment. Is it OK now? 😃